prepare/cards/xlsum.py

```diff
@@ -0,0 +1,40 @@
+from datasets import get_dataset_config_names
+
+configs = get_dataset_config_names("GEM/xlsum")  # the languages
+# now configs is the list of all languages showing in the dataset
+
+from src.unitxt.blocks import (
+    AddFields,
+    FormTask,
+    InputOutputTemplate,
+    LoadHF,
+    RenameFields,
+    TaskCard,
+    TemplatesList,
+)
+from src.unitxt.catalog import add_to_catalog
+from src.unitxt.test_utils.card import test_card
+
+langs = configs
+
+for lang in langs:
+    card = TaskCard(
+        loader=LoadHF(path="GEM/xlsum", name=lang),
+        preprocess_steps=[
+            RenameFields(field_to_field={"text": "document", "target": "summary"}),
+            AddFields(fields={"text_type": "text"}),
+        ],
+        task=FormTask(
+            inputs=["document"],
+            outputs=["summary"],
+            metrics=["metrics.rouge"],
+        ),
+        templates=TemplatesList(
+            [
+                InputOutputTemplate(input_format="{document}", output_format="{summary}"),
+            ]
+        ),
+    )
+    if lang == "english":
```

Comment:
```suggestion
    if lang == langs[0]
```

Reply:
The reason is , is that we don't want to add to catalogs cards, before at least one is tested.  And we don't know if English is the first.

Reply:
Thanks, @yoavkatz , this is a very nice thought, that did not occur to me.
Fixing right away!

Reply:
Thanks again, @yoavkatz , fixed this, and also for Amazon from yesterday.
Please see if you can approve now.